src/java.base/share/native/libjli/java.c

```diff
@@ -566,6 +567,10 @@ JavaMain(void* _args)
     jfieldID noArgMainField;
     jboolean noArgMain;
 
+#if defined(J9VM_OPT_CRAC_SUPPORT)
+    handleCRaCRestore(argc, argv);
+#endif /* defined(J9VM_OPT_CRAC_SUPPORT) */
+
```

Comment:
Previously, this call occurred after the call to `RegisterThread()`: Why is it now before?

Reply:
`handleCRaCRestore()` doesn't require `RegisterThread()`, and we would like to invoke `criu restore` as early as possible.
On a side note, I attempted to move this function into OpenJ9 respo, hoping to avoid backporting it to other extension repos. In some quick experiments on a Fyre image, it was ~30ms slower which doesn't fit the instanton perf goal.